.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,41 @@
+<!-- 
+Thanks for showing interest in filing an issue! 
+However, make sure that we do not have any duplicates already open. You can ensure 
+this by searching the issue list here https://github.com/minishift/minishift/issues.
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
+
+<!--
+If this is a BUG REPORT, please:
+  - Fill in as much of the template below as you can.  If you leave out
+    information, we can't help you as well.
+
+If this is a FEATURE REQUEST, please:
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+**Minishift version** (use `minishift version`):
+
+
+**Environment**:
+- **OS** (e.g. from /etc/os-release):
+- **VM Driver** :
+- **Others**:
+
+
+**What happened**:
+
+
+**What you expected to happen**:
+
+
+**How to reproduce it** (as minimally and precisely as possible):
+
+
+**Anything else do we need to know**:
```

Comment:
-1, in my experience you quite often need more time to clean up a issue created via the template, than it takes to ask information which is appropriate in the context. And it absolutely gets in the way when creating non bug reports.